youtube_dl/extractor/youtube.py

```diff
@@ -312,10 +312,14 @@ def extract_videos_from_page(self, page):
 class YoutubePlaylistsBaseInfoExtractor(YoutubeEntryListBaseInfoExtractor):
     def _process_page(self, content):
         for playlist_id in orderedSet(re.findall(
-                r'<h3[^>]+class="[^"]*yt-lockup-title[^"]*"[^>]*><a[^>]+href="/?playlist\?list=([0-9A-Za-z-_]{10,})"',
-                content)):
-            yield self.url_result(
-                'https://www.youtube.com/playlist?list=%s' % playlist_id, 'YoutubePlaylist')
+                r'<h3[^>]+class="[^"]*yt-lockup-title[^"]*"[^>]*><a[^>]+href="/?playlist\?list=([0-9A-Za-z-_]{10,})"', content)):
+            playlistURL = 'https://www.youtube.com/playlist?list=%s' % playlist_id
+
+            # get the data from that page and grab the title
+            playlistTitle = self._og_search_title(self._download_webpage(playlistURL, playlist_id))
```

Comment:
1. No additional requests here.
2. Will break extraction if failed.
3. Bother to read coding conventions before submitting PR.